### PR TITLE
Troubleshooting for SBT 1.0 and Coursier

### PR DESCRIPTION
**Pull request by iht**
Hi,

Recently, I had an issue with ensime-sbt, that was solved by just following the troubleshooting instructions. I did a couple of different steps, as I am using SBT 1.0, and the corrupted JAR was in the Coursier cache. I am adding those extra steps to the documentation.

Thanks in advance.

Israel

### CODE REVIEW

File: build_tools/sbt.md [hunk 1]
```diff
@@ -178,6 +178,12 @@ If **that** doesn't work, move your ivy folder aside, it has probably become cor
 mv ~/.ivy2 ~/.ivy2.bak
 ```
 
+If you are using [coursier](http://get-coursier.io), your `.coursier` folder may also be corrupted. Move it away before trying again:
```
Comment (yyadavalli) [inline build_tools/sbt.md]: The plugin uses coursier, even if the build doesn't. You can reword it to remove the `if`.

May be change the current statement to 

*If that doesn’t work, move your ivy and coursier folders aside, the caches have probably become corrupted:*

Comment (iht) [reply build_tools/sbt.md]: Done. I have just pushed a new commit.

**Comment by iht** [comment]
Thanks!
